src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.State.cs

```diff
@@ -408,6 +400,23 @@ private State()
                     lastAdditionalDependentFileTimesChangedAtUtc: lastAdditionalDependentFileTimesChangedAtUtc,
                     lastItemsChangedAtUtc: lastItemsChangedAtUtc, lastCheckedAtUtc: LastCheckedAtUtc);
 
+
+                DateTime getLastTimeAdditionalDependentFilesAddedOrRemoved(IImmutableDictionary<string, DateTime> additionalDependentFileTimes)
+                {
+                    var lastExistingAdditionalDependentFiles = AdditionalDependentFileTimes.Where(pair => pair.Value != DateTime.MinValue)
+                                                                       .Select(pair => pair.Key)
+                                                                       .ToImmutableHashSet();
+
+                    var currentExistingAdditionalDependentFiles = additionalDependentFileTimes.Where(pair => pair.Value != DateTime.MinValue)
+                                                                                              .Select(pair => pair.Key)
+                                                                                              .ToImmutableHashSet();
+
+                    bool AdditionalDependentFilesChanged = !lastExistingAdditionalDependentFiles.SetEquals(currentExistingAdditionalDependentFiles);
```

Comment:
Local variables should start with a lowercase letter.

Reply:
done